src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java

```diff
@@ -44,7 +45,8 @@ public void ImageStorage_001_deleteImage() {
 
     DeletePageObject delete = file.deletePage();
     delete.submitDeletion();
-    filesPage.getBannerNotifications().verifyNotificationMessage();
+    Assertion.assertTrue(filesPage.getBannerNotifications().isNotificationMessageVisible(),
+                         "Banner notification message is not visible");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Banner notification message is not visible" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)